app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -120,14 +125,15 @@ class TopicFragmentPresenter @Inject constructor(
   }
 
   private fun computeTabPosition(tab: TopicTab): Int {
-    return if (enableExtraTopicTabsUi.value) tab.positionWithFourTabs else tab.positionWithTwoTabs
+    return tab.getPosition(enableTopicInfoTab.value, enableTopicPracticeTab.value)
   }
 
   private fun setUpViewPager(
     viewPager2: ViewPager2,
     classroomId: String,
     topicId: String,
-    isConfigChanged: Boolean
+    isConfigChanged: Boolean,
+    enablePracticeTab: Boolean
```

Comment:
It is somewhat inconsistent to pass `enablePracticeTab` boolean as an argument, and then call `enableTopicInfoTab` from the top level declaration. the function would be cleaner if you passed both variables in the function parameter list.